test/commands/doctor.test.ts

```diff
@@ -51,35 +66,36 @@ describe('Doctor Command', () => {
   let promptStub: sinon.SinonStub;
   let openStub: sinon.SinonStub;
 
-  oclifConfigStub = fromStub(
-    stubInterface<Config>(sandbox, {
-      pjson: {
-        engines: {
-          node: 'node-v16.17.0',
-        },
+  oclifConfig = {
```

Comment:
If you use the `fromStub(stubInterface<>)` pattern you won't need to do the `as unknown as Config;`. That's the super cool thing about using those utilities.

Reply:
@shetzel That's how I started, but in the tests when I would override oclifConfig.versionDetails = {...} the change never appeared.